src/sentry/incidents/subscription_processor.py

```diff
@@ -630,8 +630,8 @@ def trigger_alert_threshold(
             if not self.active_incident:
                 detected_at = self.calculate_event_date_from_update_date(self.last_update)
                 activation: AlertRuleActivations | None = None
-                if self.alert_rule.monitor_type == AlertRuleMonitorType.ACTIVATED:
```

Comment:
was previously comparing to the enum itself rather than the `.value`

Reply:
Instead of using value, could we switch `AlertRuleMonitorType` to an [IntEnum](https://docs.python.org/3/library/enum.html#enum.IntEnum)?

Reply:
The Enum isn't quite an `IntEnum` though.
`... but its members are also integers `

the members of the enum are not ints but more descriptors of what the enum is for 🤷

Reply:
by members there I think it means the values that represent those members?

Reply:
I think the example is maybe a bit extra in that the members are ints and can be manipulated freely, but we really just care about the equality part.

Reply:
for now i'll leave it as is.

out of scope for this PR to change the enum type / value type